docs/src/pages/landing-page/Index.vue

```diff
@@ -168,21 +210,87 @@
       <q-img class="absolute-bottom" width="50%" src="~assets/landing-page/homepage-background-images/astronaut-left-hand.png" />
       <q-img width="50%" class="absolute-top-right" src="~assets/landing-page/homepage-background-images/astronaut-right-hand.png" />
     </div>
+
+    <div class="text-center lg-mb-large">
```

Comment:
I'm pretty sure `Full sponsor's list` button should bring the user to this section

Reply:
yeah, that would be correct. I'll in the next PR where I link to components page add that.